src/ripple/server/impl/BaseWSPeer.h

```diff
@@ -328,7 +388,6 @@ do_read()
     impl().ws_.async_read(op_, rb_, strand_.wrap(
         std::bind(&BaseWSPeer::on_read,
             impl().shared_from_this(), placeholders::error)));
-    cancel_timer();
```

Comment:
It really depends how paranoid you want to be. We have overlapping protections, so we're not relying on this to protect against slowloris-style attacks. Still, I'd like to ensure that the other side is reading data, and there's really no good way to do that other than demanding ping replies.